src/App.js

```diff
@@ -1,38 +1,33 @@
 import React, { Component } from 'react';
-import './App.scss';
-import { FilmsList } from './components/FilmsList';
-import { NewFilm } from './components/NewFilm';
-import { films } from './data';
-import { FormField } from './components/FormField';
 import {
   BrowserRouter,
   Switch,
   Route,
 } from 'react-router-dom';
+
+import './App.scss';
+
+import { FilmsList } from './components/FilmsList';
+import { NewFilm } from './components/NewFilm';
+import { FormField } from './components/FormField';
 import { FilmDetails } from './components/FilmDetails';
+import { store, addNewFilm } from './store';
 
 const API_URL = 'http://www.omdbapi.com/?apikey=2f4a38c9&t=';
```

Comment:
опционально
```suggestion
const API_URL = 'https://www.omdbapi.com/?apikey=2f4a38c9&t=';
```

рекоменддую почитать отличия `HTTP` & `HTTPS`

Reply:
знаком, при https идет безопасная передача данных так как используются дополнительные протоколы SSL\TLS (криптографические протоколы) асинхронного шифрования пакетов данных, которые передаем по соединению TCP :)